Ba6_MiniProjet/move.c

```diff
@@ -33,32 +35,38 @@ void set_speed(int16_t new_speed) {
 	speed = new_speed;
 }
 
-
 int16_t convert_speed(uint8_t code){
 
 	int16_t speed = 0;
 
 	if(code > 25){
 		speed = (MAX_SPEED - MIN_SPEED)/13*code+3*MIN_SPEED-2*MAX_SPEED; //vitesse entre 20 et 100%
 	}else{
-		speed = (-MAX_SPEED + MIN_SPEED)/12*code+2*MAX_SPEED-MIN_SPEED; //vitesse entre -20 et -100%
+		speed = -( (-MAX_SPEED + MIN_SPEED)/12*code+2*MAX_SPEED-MIN_SPEED); //vitesse entre -20 et -100%
```

Comment:
TO DO : add explaination

Reply:
sinon c'est pas -, la vitesse est positive, donc on l'a rend négative